### PR TITLE
Add files for noovex

### PR DESCRIPTION
**Pull request by Freezyii2**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/noovex/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/noovex/background@2x.png:1]: Compress image

File: minecraft_servers/noovex/background.png
Comment (DoJapHD) [inline minecraft_servers/noovex/background.png:1]: Compress image

File: minecraft_servers/noovex/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "server_name": "noovex",
+  "nice_name": "Noovex",
+  "direct_ip": "beta.noovex.net",
+  "server_wildcards": [
+    "%.noovex.net"
```
Comment (DoJapHD) [inline minecraft_servers/noovex/manifest.json:6]: Not a real wildcard
https://i.imgur.com/a6sf0IJ.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/noovex/background.png
Comment (maaaathis) [inline minecraft_servers/noovex/background.png:1]: Compress this file

File: minecraft_servers/noovex/background@2x.png
Comment (maaaathis) [inline minecraft_servers/noovex/background@2x.png:1]: Compress this file

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/noovex/background@2x.png
Comment (Freezyii2) [reply minecraft_servers/noovex/background@2x.png:1]: Hey, I changed it again. Sorry for the inconvenience, this is my first time doing this

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**